services/movie_session.py

```diff
@@ -0,0 +1,52 @@
+from db.models import Movie, CinemaHall, MovieSession
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
```

Comment:
Consider using `CinemaHall.objects.filter(id=cinema_hall_id).first()` to avoid `DoesNotExist` exceptions if the cinema hall with the specified ID is not found. This will return `None` instead of raising an exception.